deploy/olm-catalog/kubefed-operator/4.2/image-references

```diff
@@ -0,0 +1,12 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: ose-kubefed
+    from:
+      kind: DockerImage
+      name: quay.io/kubernetes-multicluster/kubefed
```

Comment:
cc @abhat 

Are the names here (`ose-*`) correct?

Reply:
Yes, from my conversation with @shawn-hurley the name field should be the name in ose for your image. and from.name should be the full image that is currently in the CSV. We technically don't need the first tag at all since our kubefed (operand) is not in the CSV.

Reply:
Fixed.